.env.sample

```diff
@@ -59,6 +59,7 @@ DISABLE_LEARNER_SERVICE_ON_OFF = ""
 KAFKA_COMMUNICATIONS_ON_OFF = ""
 KAFKA_URL = "" // IP address of kafka server with port without HTTP
 NOTIFICATIONS_TOPIC = "OFF/TOPIC_NAME"
+KAFKA_GROUP_ID = ""
```

Comment:
@aman-tunerlabs - Can you add it envVariablesHelper so that it is mandatory value?